examples/golang-todo-rest-crud/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Chelodoz
```

Comment:
We can remove this file in order to use the LICENSE of the repository 😊 you will appear on the README as a contributor as soon as this PR is merged 👌🏻

Reply:
✔️ Done!